tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1092,6 +1092,7 @@ private static JSONObject createJSONLocMap(SurveyMain sm) throws JSONException {
         // locales will have info about each locale, including name
         JSONObject locales = new JSONObject();
         SupplementalDataInfo sdi = sm.getSupplementalDataInfo();
+        final StandardCodes sc = StandardCodes.make();
```

Comment:
is this "sc" unused? I see no reference to it in the changes

Reply:
good catch. replaced by SurveyMain.getLocales()